src/ResourceManager/MachineLearning/Commands.MachineLearning/Microsoft.Azure.Commands.MachineLearning.dll-Help.psd1

```diff
@@ -9,7 +9,7 @@
 @{  
   
 # Version number of this module.  
-ModuleVersion = '0.9.0' 
+ModuleVersion = '1.0.0' 
```

Comment:
@NonStatic2014 revert these changes as we will bump the version during the release process